msibi/tutorials/propane/propane.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Propane Tutorial\n",
+    "\n",
+    "Created by Davy Yue 2017-06-14\n",
+    "\n",
+    "\n",
+    "### Import packages\n",
+    "\n",
+    "According to PEP8 standards, imports are grouped into \n",
+    "1. standard library imports \n",
+    "2. related third party imports \n",
+    "3. local application/library specific imports\n",
+    "\n",
+    "Group 1 includes `itertools`, `string`, and `os`.\n",
+    "\n",
+    "Group 2 includes `numpy`, which is stored in the keyword `np` and can be accessed with `np.[function]`.\n",
+    "\n",
+    "Group 3 includes `MSIBI`, `State`, `Pair`, and `mie` from the `msibi` package, which manages pair-potential derivation using MultiState Iterative Boltzmann Inversion (MS-IBI).\n"
+   ]
```

Comment:
You can probably just change this cell to say "Imports," or just leave it out altogether.